.github/workflows/CI-distro.yml

```
@@ -0,0 +1,115 @@
+          - gap-package: 'gradedringforhomalg'        # `Error, found no Singular executable in PATH while searching the following list` 
+          - gap-package: 'guarana'                    # `AL_EXECUTABLE, the executable for PARI/GP, has to be set`
+          #- gap-package: 'guava'                      # test failure
+          - gap-package: 'hap'                        # `polymake command not found. Please set POLYMAKE_COMMAND by hand`
+          - gap-package: 'hapcryst'                   # `polymake command not found. Please set POLYMAKE_COMMAND by hand`
+          #- gap-package: 'io'                         # test failure
+          - gap-package: 'itc'                        # dependency `xgap` has no jll
+          - gap-package: 'jupyterkernel'              # no jll
+          - gap-package: 'localizeringforhomalg'      # `Error, found no Singular executable in PATH while searching the following list`'
+          - gap-package: 'modulepresentationsforcap'  # `Error, found no Singular executable in PATH while searching the following list`'
+          - gap-package: 'normalizinterface'          # `Error, Some error in the normaliz input data detected: Cone Property in last line not allowed for lattice ideals`
+          #- gap-package: 'packagemanager'             # test failure
+          - gap-package: 'polenta'                    # `AL_EXECUTABLE, the executable for PARI/GP, has to be set`
+          - gap-package: 'polycyclic'                 # ???
+          - gap-package: 'polymaking'                 # `polymake command not found. Please set POLYMAKE_COMMAND by hand`
+          - gap-package: 'profiling'                  # segfaults during testing
+          - gap-package: 'radiroot'                   # `AL_EXECUTABLE, the executable for PARI/GP, has to be set`
+          - gap-package: 'ringsforhomalg'             # `Error, found no Singular executable in PATH while searching the following list`'
+          - gap-package: 'semigroups'                 # no jll
+          - gap-package: 'singular'                   # `Error, Singular executable file not found!`
+          - gap-package: 'toricvarieties'             # `Error, found no Singular executable in PATH while searching the following list`'
+          #- gap-package: 'typeset'                    # `ReadPackage could not read <digraphs>/gap/doc.g`
+          #- gap-package: 'wpe'                        # ???
+          - gap-package: 'xgap'                       # no jll
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: "Set up Julia"
+        uses: julia-actions/setup-julia@v2
+        with:
+          version: ${{ matrix.julia-version }}
+          arch: ${{ matrix.julia-arch }}
+      - name: "Cache artifacts"
+        uses: julia-actions/cache@v2
+        with:
+          include-matrix: false # don't create a cache per GAP package
```

```suggestion
          include-matrix: false # don't create a cache per GAP package
          cache-scratchspaces: false
```
Let's try to re-enable this setting once https://github.com/oscar-system/GAP.jl/pull/1100 is merged.